common/lib/xmodule/xmodule/js/src/problem/edit.js

```diff
@@ -560,50 +560,91 @@
                     // Line split here, trim off leading xxx= in each function
                     var answersList = p.split('\n'),
 
-                        processNumericalResponse = function(val) {
-                            var params, answer, string, textHint, hintLine, value;
-                            // Numeric case is just a plain leading = with a single answer
-                            value = val.replace(/^\=\s*/, '');
+                        processNumericalResponse = function (answerValues) {
+                            var firstAnswer, numericalResponseString, additionalAnswerString, textHint, hintLine;
 
-                            textHint = extractHint(value);
+                            isRangeToleranceCase = function (answer) {
+                                return _.contains(['[', '('], answer[0]) && _.contains([']', ')'], answer[answer.length - 1])
+                            }
+
+                            getAnswerData = function (answerValue) {
+                                var answerData = {},
+                                    answerParams = /(.*?)\+\-\s*(.*?$)/.exec(answerValue);
+                                if (answerParams) {
+                                    answerData.answer = answerParams[1].replace(/\s+/g, ''); // support inputs like 5*2 +- 10
+                                    answerData.default = answerParams[2];
+                                } else {
+                                    answerData.answer = answerValue.replace(/\s+/g, ''); // support inputs like 5*2
+                                }
+                                return answerData;
+                            }
+
+                            // First string case is s?= [e.g. = 100]
+                            firstAnswer = answerValues[0];
+                            firstAnswer = firstAnswer.replace(/^\=\s*/, '');
```

Comment:
I still think we can use this `firstAnswer  = answerValues[0].replace(/^\=\s*/, '');` or i'm missing something ?